eduNEXT/tvm
# perf: tvm migrate commands

Description
========
This PR migrates some commands to clean architecture (hexagon architecture), the commands are:
- list_versions
- install
- uninstall
- use
- install_plugin
- uninstall_plugin

The unit test was made only for commands above

How to test:
=========
1. Use this branch (perf/tvm_migrate_commands)
2. Create and activate a virtual environment
3. Run commands of the description, Examples:
```bash
tvm list
tvm install v13.3.1
tvm uninstall v13.3.1
tvm use v13.3.1
tvm plugins install tutor-mfe
tvm plugins uninstall tutor-mfe
```
5. Run ``make run_tests`` to verify the unit tests.

## Checklist for Merge

- [x] Tested in a local environment
- [ ] At less an approval
